sonar-db/src/main/resources/org/sonar/db/user/UserMapper.xml

```diff
@@ -98,6 +98,14 @@
     </where>
   </select>
 
+  <select id="countByEmail" parameterType="String" resultType="long">
+    SELECT count(u.id)
+    FROM users u
+    <where>
```

Comment:
I know you like using Mybatis tag `<where>` but `where  u.email=...` sounds enough here :-)